src/Controls/tests/TestCases.HostApp/Issues/Issue2354.cs

```diff
@@ -8,16 +8,16 @@ protected override void Init()
 		{
 			var presidents = new List<President>();
 
-			presidents.Add(new President($"Presidente 44", 1, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/Fruits.jpg?raw=true"));
-			presidents.Add(new President($"Presidente 43", 2, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/person.png?raw=true"));
-			presidents.Add(new President($"Presidente 42", 3, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/photo.jpg?raw=true"));
-			presidents.Add(new President($"Presidente 41", 4, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/FlowerBuds.jpg?raw=true"));
-			presidents.Add(new President($"Presidente 40", 5, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/games.png?raw=true"));
-			presidents.Add(new President($"Presidente 39", 6, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/gear.png?raw=true"));
-			presidents.Add(new President($"Presidente 38", 7, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/coffee.png?raw=true"));
-			presidents.Add(new President($"Presidente 37", 8, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/xamarinstore.jpg?raw=true"));
-			presidents.Add(new President($"Presidente 36", 9, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/oasis.jpg?raw=true"));
-			presidents.Add(new President($"Presidente 35", 10, $"https://github.com/dotnet/maui/blob/main/src/Compatibility/ControlGallery/src/Android/Resources/drawable/Vegetables.jpg?raw=true"));
+			presidents.Add(new President($"Presidente 44", 1, $"https://github.com/dotnet/maui/blob/main/src/Controls/tests/TestCases.HostApp/Resources/Images/avatar.png?raw=true"));
```

Comment:
We removed the Legacy Gallery https://github.com/dotnet/maui/pull/26068 and the test was using images from that gallery. For that reason, was failing.